client/lib/query-manager/index.js

```diff
@@ -104,6 +104,27 @@ export default class QueryManager {
 	}
 
 	/**
+	 * A sorting function that defines the sort order of items under
+	 * consideration of the specified query.
+	 *
+	 * @param  {Array}  keys  Keys to be sorted
+	 * @param  {Array}  items Items by which to sort
+	 * @param  {Object} query Query object
+	 * @return {Array}        Sorted keys
+	 */
+	sort( keys, items, query ) {
+		return keys.sort( ( keyA, keyB ) => {
```

Comment:
> It's a shame there's no non-mutative way to sort an array, not even in Lodash*!

Yeah, I've searched hi and lo-dash before arriving at this impl. It's a shame that JS doesn't mandate the 0 return to work for comparators in the first place, as that clearly rules out any kind of "default", un-mutating comparator. Given that fact, maybe we should add a comment to https://github.com/lodash/lodash/issues/2090 asking for a comparator after all...

Dropping the return for now.